Modules/_ctypes/_ctypes.c

```diff
@@ -4813,8 +4823,8 @@ PyTypeObject PyCArray_Type = {
     0,                                          /* tp_getattro */
     0,                                          /* tp_setattro */
     &PyCData_as_buffer,                         /* tp_as_buffer */
-    Py_TPFLAGS_DEFAULT | Py_TPFLAGS_BASETYPE, /* tp_flags */
-    PyDoc_STR("XXX to be provided"),            /* tp_doc */
+    Py_TPFLAGS_DEFAULT | Py_TPFLAGS_BASETYPE,   /* tp_flags */
+    PyDoc_STR(array_doc),                       /* tp_doc */
```

Comment:
We can directly use this variable.
```suggestion
    array_doc,                                  /* tp_doc */
```

Reply:
Ah okay, fair enough. That makes sense.